### PR TITLE
Track Session Creation Time

### PR DESCRIPTION
**Pull request by oliviajain**
**Description**: Monitor session creation time.

**Motivation and Context**
- Be able to catch regressions in session creation time.

### CODE REVIEW

File: onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh [hunk 1]
```diff
@@ -8,11 +8,11 @@ then
     git remote update
     if [ "$1" = "8.0" ]
     then 
-        git checkout "$1"'-GA'
+        git checkout 1f416bb
```
Comment (jywu-msft) [inline onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh:11]: why this specific commit?

Comment (oliviajain) [reply onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh:11]: 8.0-GA causes a build failure, Chi told me this to run with this commit, which made ORT able to build.

Comment (jywu-msft) [reply onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh:11]: that looks odd. 8.0-GA worked fine when we were developing TRT EP support for 8.0. is it an issue with the latest onnxruntime master? does the rel-1.10.0 branch still work? @chilo-ms can you provide additional context about this?

Comment (chilo-ms) [reply onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh]: I'm using the branch [trt8debug](https://github.com/microsoft/onnxruntime/tree/trt8debug/cmake/external) to compare and debug regression on trt8.2. The onnx-tensorrt uses that commit.

Comment (oliviajain) [reply onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh]: Confirmed, 8.0-GA does work, sending an email with more information regarding master shortly.

File: dockerfiles/Dockerfile.tensorrt [hunk 2]
```diff
@@ -26,8 +26,8 @@ RUN git clone --single-branch --branch ${ONNXRUNTIME_BRANCH} --recursive ${ONNXR
 
 # Checkout appropriate TRT_VERSION and build
 RUN cd onnxruntime &&\
-    trt_version=${TRT_VERSION%.*.*} &&\
+    trt_version=${TRT_VERSION:0:3} &&\
```
Comment (oliviajain) [inline dockerfiles/Dockerfile.tensorrt:29]: a bit more info: by default, Nvidia containers already have a TRT_VERSION environment variable, meaning the one we pass in is useless

21.07: 8.0.1.6+cuda11.3.1.005

File: onnxruntime/python/tools/tensorrt/perf/post.py [hunk 3]
```diff
@@ -115,6 +123,9 @@ def get_time():
     date_time = time.strftime(time_string_format)
     return date_time
 
+def get_identifier(commit_id, trt_version, branch):
```
Comment (oliviajain) [inline onnxruntime/python/tools/tensorrt/perf/post.py:126]: Using a string identifier of commit_id + trt_version + branch

**Review by jywu-msft — APPROVED** [review]

**Pull request merged**